src/Security/OAuthUserProvider.php

```diff
@@ -64,11 +81,9 @@ private function loadDatabaseUser(string $name, string $username, string $email)
             $user = $repository->findOneByEmail($email);
```

Comment:
Invert the is_null to have an early exit
```if ($user instanceof UserInterface) {
    return $user;
}
```

Reply:
Fixed